chat-plugins/scavengers.js

```diff
@@ -1373,6 +1373,46 @@ let commands = {
 		this.privateModAction(`(${user.name} has ${(change > 0 ? 'given' : 'taken')} one infraction point ${(change > 0 ? 'to' : 'from')} '${targetId}'.)`);
 		this.modlog(`SCAV ${this.cmd.toUpperCase()}`, user);
 	},
+
+	modsettings: {
+		'': 'update',
+		'update': function (target, room, user) {
+			if (!this.can('declare') || room.id !== 'scavengers') return false;
```

Comment:
does this need to check global perms?

Reply:
oh oops havent touched ps code in a looong time - ill fix when i get on a computer